packages/components/src/organisms/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, FunctionComponent } from 'react'
+
+import { ImageGallerySelector } from '../..'
+
+type ImageComponentType = FunctionComponent<{
+  url: string
+  alternateName?: string
+  loading?: 'eager' | 'lazy'
+  onLoad?: (image: any) => void
+}>
+
+export type ImageElementData = {
```

Comment:
Hmm, `react-docgen-typescript` kind of work better with interface for automatic PropsSection, Maybe we can test using interface here and changing in the docs?